client/src/pages/RegistrationPage.jsx

```diff
@@ -0,0 +1,221 @@
+                  </span>
+                )}
+              </div>
+
+              {touched.name && errors.name && (
+                <p className="help is-danger">{errors.name}</p>
+              )}
+            </div>
+            <div className="field">
+              <label htmlFor="email" className="label">
+                Email
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validateEmail}
+                  name="email"
+                  type="email"
+                  id="email"
+                  placeholder="e.g. bobsmith@gmail.com"
+                  className={cn('input', {
+                    'is-danger': touched.email && errors.email,
+                  })}
+                />
+
+                <span className="icon is-small is-left">
+                  <i className="fa fa-envelope"></i>
+                </span>
+
+                {touched.email && errors.email && (
+                  <span className="icon is-small is-right has-text-danger">
+                    <i className="fas fa-exclamation-triangle"></i>
+                  </span>
+                )}
+              </div>
+
+              {touched.email && errors.email && (
+                <p className="help is-danger">{errors.email}</p>
+              )}
+            </div>
+            <div className="field">
+              <label htmlFor="password" className="label">
+                Password
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validatePassword}
+                  name="password"
+                  type="password"
+                  id="password"
+                  placeholder="*******"
+                  className={cn('input', {
+                    'is-danger': touched.password && errors.password,
+                  })}
+                />
+
+                <span className="icon is-small is-left">
+                  <i className="fa fa-lock"></i>
+                </span>
+
+                {touched.password && errors.password && (
+                  <span className="icon is-small is-right has-text-danger">
+                    <i className="fas fa-exclamation-triangle"></i>
+                  </span>
+                )}
+              </div>
+
+              {touched.password && errors.password ? (
+                <p className="help is-danger">{errors.password}</p>
+              ) : (
+                <p className="help">At least 6 characters</p>
+              )}
+            </div>
+            <div className="field">
+              <button
+                type="submit"
+                className={cn('button is-success has-text-weight-bold', {
+                  'is-loading': isSubmitting,
+                })}
+                disabled={
+                  isSubmitting || error.name || errors.email || errors.password
```

Comment:
There is a potential issue here. The condition `error.name` should likely be `errors.name` to match the structure of the errors object. This could prevent the submit button from being disabled correctly.